build.gradle.kts

```diff
@@ -29,18 +29,25 @@ repositories {
     maven("https://plugins.gradle.org/m2")
 }
 
+configurations.all {
+    // Dependencies provided by IntelliJ runtime
+    exclude("com.google.code.gson", "gson")
+    exclude("org.slf4j", "slf4j-api")
+}
+
 dependencies {
-    implementation("io.pebbletemplates:pebble:3.1.5")
-    implementation("org.junit.jupiter:junit-jupiter:5.8.1")
-    detektPlugins("io.gitlab.arturbosch.detekt:detekt-formatting:1.10.0-RC1")
+    implementation("com.github.doyaaaaaken:kotlin-csv-jvm:1.6.0") {
+        // Not using any of the async functionality
+        exclude("org.jetbrains.kotlinx", "kotlinx-coroutines-core")
```

Comment:
The CSV reader used by LD plugin doesn't touch any of the asynchronous functionality of the library, so I believe coroutines can be excluded. Unfortunately I can't verify this with Code References feature, but I tried reading some irrelevant CSV file using the same method elsewhere and that worked just fine.

This saves around 1.4Mb.